templates/admin/action-buttons.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/** @var WC_Gateway_Swedbank_Pay_Cc $gateway */
+/** @var WC_Order $order */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+} // Exit if accessed directly
+
+?>
+
+<?php if ( $gateway->core->canCapture( $order->get_id() ) ) : ?>
```

Comment:
Shouldn't this be called `can_capture` for consistency?

```suggestion
<?php if ( $gateway->core->can_capture( $order->get_id() ) ) : ?>
```

Reply:
@asbjornu No, the core library have different methods and code style because it's more flexible. Wordpress (and WooCommerce) uses the Snake case. But the core library uses Camel case because it can be used for another shopping solutions.